File: hw/cevallos/u1/hw14SaveChickenFarmInJson/python/egg.py
class Egg:
    def __init__(self, size):
        self.size = size
    
    def __str__(self):
        return f"Egg{{size={self.size}}}"
    
    def __repr__(self):
        return self.__str__()
    
    def get_size(self):
        return self.size
    
    def set_size(self, size):
        self.size = size

File: hw/cevallos/u1/hw14SaveChickenFarmInJson/Java/src/model/Egg.java
package model;

/**
 *
 * @author Mateo Cevallos
 */
public class Egg {
    private char size;

    @Override
    public String toString(){
        return "Egg{" + "size=" + size + '}';
    }
    
    public Egg(char size){
        this.size = size;
    }
    
    public char getSize(){
        return size;
    }
    
    public void setSize(char size){
        this.size = size;
    }
}

